### ISSUE TITLE
Microsoft 365 Learning Pathways technical difficulties

### ISSUE THREAD

**Issue by jhwang-ms** [pnp/custom-learning-office-365#786]
Hello all. My name is Jessie Hwang and I'm on the content management team. I want to accept and acknowledge that there have been challenges about updated content on M365 Learning Pathways. We've had some technical difficulties, as you've seen with the videos and the changing of the templates, and we are working to get these issues resolved before we publish new content. I will keep this issue updated while we're working through these technical problems. Next update expected date: +7 days.

**Comment by clarelcampbell** [comment]
Morning - thanks for the update and appreciate your efforts to get these issues fixed.

**Comment by jhwang-ms** [comment]
We are still working through escalations, and I hope to have a better update next week.

**Comment by USCloud-EricWilborn** [comment]
Good morning. Is there any new status on when updates can be expected to be pushed for M365 Learning Pathways?

**Comment by jhwang-ms** [comment]
Hello everyone. We are still working through the escalation chain and I do not have better news at this time. The engineering leadership team who needs to know this is aware and they, alongside our leadership team, are working to find the right solutions to untangle these issues. Again, I appreciate everyone's patience and grace while I continue to chase things up the chain.

**Comment by KaveSys** [comment]
Hello everyone, I don't want to waste your time to reply on my comment, but can you explain which are these "technical difficulties" and if there is a sort of "ETA" on that?
Is a nice solution for end users and administrator, help really the users to have a great experience on MS products and services.

Thanks anyway for the work that you and the teams involved are doing 🤜🤛

**Comment by Jordan-WSECU** [comment]
Thank you for the follow-up, Jessie. Much appreciated! 🙂


Jordan Hughes |  Book a meeting with Jordan ***@***.***?anonymous&ep=pcard>

Sr. Software Engineer - SharePoint and Automation

PO Box WSECU, Olympia, WA 98507

***@***.******@***.***>

www.wsecu.org<http://www.wsecu.org/>

800.562.0999 x10502



________________________________

**Comment by jhwang-ms** [comment]
Hi everyone - first of all, thank you for your patience and grace while we've been figuring out root causes, escalating to engineering teams and leadership to try to find solutions. A quick update: we have identified a couple of possible solutions and are testing them to ensure they work (and not fix one thing just to create 5 more issues). I'll be back with another update as soon as I get one from our engineering team. Thank you.

**Comment by Scu-OSN** [comment]
Many thanks for your ongoing commitment, Jessie. Much appreciated!

**Comment by TomMeadows1990** [comment]
Hi Jessie, 

I wondered if there was any further updates on when this issue may be resolved? I'm launching some pages for colleagues shortly and these really useful webparts are a big part of them, so I was hoping the issue may have been resolved by now. 

Any update would be appreciated. 

many thanks,
Tom

**Comment by jhwang-ms** [comment]
Hi everyone - unfortunately, my update is that I do not have an update at this time. I'm told that this is getting worked on above my head but I do not have any additional information to share. I will continue to ask and ask and ask my manager and the leadership chain above me for updates and ETAs.

**Comment by Rosie-EAH** [comment]
@jhwang-ms Hi Jessie - this doesn't include the issue raised by @TomMeadows1990, https://github.com/pnp/custom-learning-office-365/issues/771#issuecomment-2051985168, which we're also struggling with.

This issue has the potential to stop us using the learning pathways entirely, and I can only see this thread as the source of why the pages are behaving this way. https://github.com/pnp/custom-learning-office-365/issues/728#issue-1646858987 

Can you advise how we can address this please? Thanks.

**Comment by clarelcampbell** [comment]
Hi @jhwang-ms not so much an issue but a format change I have noticed this morning. Pages we have made ourselves and then pull into Microsoft 365 Learning Pathways (MLP) using the URL to create the asset now show the page banner. This takes up space in the window unfortunately so not the best user experience and for pages only used in MLP we haven't added a picture so now will need to go back through our own pages (which we have a lot of due to no MLP Microsoft content updates since 2023) to edit. I cant see this update listed on MLP Microsoft website pages and as a site owner I haven't been notified. Along with the many issues this ticket addresses, please can you consider the communication with the admins who manage the MLP sites - I feel this could be greatly improved.

**Comment by Rosie-EAH** [comment]
> Hi @jhwang-ms not so much an issue but a format change I have noticed this morning. Pages we have made ourselves and then pull into Microsoft 365 Learning Pathways (MLP) using the URL to create the asset now show the page banner. This takes up space in the window unfortunately so not the best user experience and for pages only used in MLP we haven't added a picture so now will need to go back through our own pages (which we have a lot of due to no MLP Microsoft content updates since 2023) to edit. I cant see this update listed on MLP Microsoft website pages and as a site owner I haven't been notified. Along with the many issues this ticket addresses, please can you consider the communication with the admins who manage the MLP sites - I feel this could be greatly improved.
> 
> ![image](https://private-user-images.githubusercontent.com/123638166/350906953-8a768ba0-6dab-40a0-ae79-72f006cf4063.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE2Mzk4NzMsIm5iZiI6MTcyMTYzOTU3MywicGF0aCI6Ii8xMjM2MzgxNjYvMzUwOTA2OTUzLThhNzY4YmEwLTZkYWItNDBhMC1hZTc5LTcyZjAwNmNmNDA2My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMlQwOTEyNTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZjJmOTRiM2FlMmI1M2UwZTNkMmNiZThmN2VjMWI1ZjUzMDAzY2NhZTU2ZTgyMWVjM2U1Y2I3OTIyZjdhZmI1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.rGIiL07pmZNP90syoal02OA35IX7jFKMQu94DoYO7IQ)

I noticed this issue this morning, whilst testing out how to resolve the page height issue.  I'm very early on with working with MLP, and somewhat relieved that it's not just me having these challenges!

**Comment by dcashpeterson** [comment]
> #728 (comment)

@Rosie-EAH the page height issue is not connected to the technical issues with publishing content. The page height issue is one that has to do with Iframed content. We do not have a technical way to tell how tall the content of one of the content pages is either from Microsoft provided content or from your own custom content. Because of this we need to set the height of the viewer to the tallest asset which currently lives in Excel. The V5 beta did shorten that height a bit but it is still set to 9000px.

**Comment by dcashpeterson** [comment]
> Hi @jhwang-ms not so much an issue but a format change I have noticed this morning. Pages we have made ourselves and then pull into Microsoft 365 Learning Pathways (MLP) using the URL to create the asset now show the page banner. This takes up space in the window unfortunately so not the best user experience and for pages only used in MLP we haven't added a picture so now will need to go back through our own pages (which we have a lot of due to no MLP Microsoft content updates since 2023) to edit. I cant see this update listed on MLP Microsoft website pages and as a site owner I haven't been notified. Along with the many issues this ticket addresses, please can you consider the communication with the admins who manage the MLP sites - I feel this could be greatly improved.
> 
> ![image](https://private-user-images.githubusercontent.com/123638166/350906953-8a768ba0-6dab-40a0-ae79-72f006cf4063.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE2NTAxMjksIm5iZiI6MTcyMTY0OTgyOSwicGF0aCI6Ii8xMjM2MzgxNjYvMzUwOTA2OTUzLThhNzY4YmEwLTZkYWItNDBhMC1hZTc5LTcyZjAwNmNmNDA2My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMlQxMjAzNDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xY2I4YjIwZjhlN2M4ZWE2MWM2N2YzODg0OTk3YjAzODRlM2M0YTkxZjk5Y2I5YzJmMGIxMTJhNDcyMmI2MWEyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.qGvZ4UzBd9HyTv5BvAvFPTnCKNDbsNM4m-T-rQPya00)

@clarelcampbell this is not related to the content publish issues. This is a change that Microsoft made to the page header. you can get more information #807 but is an issue out of our control.

**Comment by dcashpeterson** [comment]
@KaveSys and @Rosie-EAH, This issue is related to the publishing issues of new content and the template accessibility. @jhwang-ms is working on those internally at Microsoft. The page length and extra scroll is not something she has control over, that is my responsibility. 

As I mentioned above, the default height of the content area is 12,000px in V4 and in V5 in 9,000px. We need to keep this height fixed as it accommodates the longest content without introducing additional scroll bars. Introducing a scroll bar in content area has received very poor user feedback in the past and introduces user experience issues for IOS users. 

Unfortunately, we do not have any control or influence on the height of the source content as this content is used across many different environments so we can't break the content up. We understand that this is a challenge for users but it is a decision we needed to make based on the technology we have and user feedback.

**Comment by KaveSys** [comment]
@dcashpeterson everything clear. But why on V4.3 the issue regarding the height was not come up and more or less everything was good? 99% of the content is under 1000px of height. There is no way to fit automatically the content area not based on the page but based on the content itself?

It's clear that introducing a scroll bar in content area has received very poor user feedback in the past, but for sure even with this workaround, the feedback will certainly not be positive.

I know that this issue is for new content and the template accessibility.

As always thanks for the feedback and hard working you have done since now anyway!

**Comment by Rosie-EAH** [comment]
> @dcashpeterson everything clear. But why on V4.3 the issue regarding the height was not come up and more or less everything was good? 99% of the content is under 1000px of height. There is no way to fit automatically the content area not based on the page but based on the content itself?
> 
> It's clear that introducing a scroll bar in content area has received very poor user feedback in the past, but for sure even with this workaround, the feedback will certainly not be positive.
> 
> I know that this issue is for new content and the template accessibility.
> 
> As always thanks for the feedback and hard working you have done since now anyway!

We've unfortunately had to take the decision to move away from MLP due to these issues.  We simply can't put the site in front of our business users and expect a positive response with the user experience being as it is with the excessively long pages. It's such a shame because the content is good, but the user experience is unacceptable, and that's the dealbreaker for us.

I hope that it can be resolved at a point in the future.

**Comment by TomMeadows1990** [comment]
> > @dcashpeterson everything clear. But why on V4.3 the issue regarding the height was not come up and more or less everything was good? 99% of the content is under 1000px of height. There is no way to fit automatically the content area not based on the page but based on the content itself?
> > It's clear that introducing a scroll bar in content area has received very poor user feedback in the past, but for sure even with this workaround, the feedback will certainly not be positive.
> > I know that this issue is for new content and the template accessibility.
> > As always thanks for the feedback and hard working you have done since now anyway!
> 
> We've unfortunately had to take the decision to move away from MLP due to these issues. We simply can't put the site in front of our business users and expect a positive response with the user experience being as it is with the excessively long pages. It's such a shame because the content is good, but the user experience is unacceptable, and that's the dealbreaker for us.
> 
> I hope that it can be resolved at a point in the future.

Thank you for the update and for outlining the issue you face with this @dcashpeterson. 

Unfortunately we're going to have to consider removing these from our internal web pages too. The pages are accessible to thousands of our colleagues and postgraduate researchers, and unfortunately the huge swathes of grey on the pages make for a poor user experience and look unprofessional. It's a real shame as the content is useful.

**Comment by dcashpeterson** [comment]
@TomMeadows1990 @Rosie-EAH @KaveSys While it is not going to solve the core issue, in V5 I am working on a feature to allow you to always open the assets in a full screen dialog. Essentially when you select this configuration and select an asset it will open in a dialog that covers the entire window. The asset pages are still 9000px but it isn't as noticeable in the dialog. Users can close the dialog and then continue with their task. Here is a screen shot. This will be part of the V5 release coming out in early September.

![image](https://github.com/user-attachments/assets/0c173afc-083d-4151-b1b2-be81e98888a6)

**Comment by Rosie-EAH** [comment]
@dcashpeterson Thanks for the update Derek.  Would this also address the issue that the interactions on our own local sites are effectively lost - as most users will give up scrolling before they get there?  The interactions are important for measurement (i.e. likes) and for the users to bookmark the content to return to later.  At the moment, they're completely lost at the end of the dead space. Thanks.

https://github.com/user-attachments/assets/05e4c5fe-d862-463d-ae74-02172838220b

**Comment by jhwang-ms** [comment]
Hi everyone - I spoke with my manager yesterday and unfortunately, we still don't have a resolution on these issues and are still blocked on content updates. I understand many of you will have to disable this from your internal systems and that is understandable. I will convey this back to my manager, and I hope to be able to have some _good_ news soon.

**Comment by Rosie-EAH** [comment]
> > @KaveSys and @Rosie-EAH, This issue is related to the publishing issues of new content and the template accessibility. @jhwang-ms is working on those internally at Microsoft. The page length and extra scroll is not something she has control over, that is my responsibility.
> > As I mentioned above, the default height of the content area is 12,000px in V4 and in V5 in 9,000px. We need to keep this height fixed as it accommodates the longest content without introducing additional scroll bars. Introducing a scroll bar in content area has received very poor user feedback in the past and introduces user experience issues for IOS users.
> > Unfortunately, we do not have any control or influence on the height of the source content as this content is used across many different environments so we can't break the content up. We understand that this is a challenge for users but it is a decision we needed to make based on the technology we have and user feedback.
> 
> While I understand the issue with allowing for longer content without scrollbars, when integrating a MLP component on a sharepoint page, it makes the whole page unusable and gives it a broken feel.
> 
> If we're working with the axiom of the embedding size being fixed, could you consider making the size either configurable by the editor when embedding, or calculating the max size of the content for each component and making the frame embedding be the maximum size needed for this specific component.
> 
> That still wouldn't be a good solution, but it would be a more paletable intermediary solution

My colleague posted a similar suggestion a little while back: https://github.com/pnp/custom-learning-office-365/issues/820

**Comment by dcashpeterson** [comment]
@Rosie-EAH @KaveSys I have talked with the team about this enhancement request in #820 and I will be working on this as time allows now that V5 is out. You can follow #820 for progress on the extra long pages.

**Comment by clarelcampbell** [comment]
> @Rosie-EAH @KaveSys I have talked with the team about this enhancement request in #820 and I will be working on this as time allows now that V5 is out. You can follow #820 for progress on the extra long pages.

Hi @dcashpeterson - can I pick up on your ref to V5, is this a new version? will there be details on here: https://learn.microsoft.com/en-us/office365/customlearning/custom_featureupdates thanks

**Comment by dcashpeterson** [comment]
@clarelcampbell I defer that to @jhwang-ms but yes that should be updated to include the new release.

**Comment by TracyGH** [comment]
Am I reading this correctly? There have not been any content updates since September of 2023? 
Is there any hope that this effort will be prioritized and we'll get some content updates in the near future? 

![image](https://github.com/user-attachments/assets/d21a1420-119c-4dbc-b2ed-b95cd22d69b4)

**Comment by jhwang-ms** [comment]
I know folks are probably tired of hearing about non-updates, but we are currently still blocked on the technical side of things and while we've been escalating to engineering, that has not gotten much traction. While we are still trying to chase people down, our team is currently trying to figure out a long-term workaround to the multiple issues we've been experiencing. I don't have a plan to share yet but know that we have been continuously trying to fix the multitude of issues, including content updates.

**Comment by Jordan-WSECU** [comment]
Perhaps sharing the nature of the technical challenges with the community could lead to useful suggestions. The likelihood of such a crowdsourcing approach being successful would probably depend on providing both details regarding the specific issues, but also some guidelines concerning the kinds of possible solutions that (a) might be workable; and (b) have already been considered and rejected.



Jordan Hughes

________________________________

**Comment by 4Ilan** [comment]
The concept of the Microsoft 365 Learning Pathways is nice, but it is concerning that it hasn't been updated since September of 2023. I am almost inclining of creating our own playlists from sites such as https://support.microsoft.com/en-us/office/outlook-training-8a5b816d-9052-4190-a5eb-494512343cca and https://support.microsoft.com/en-us/office/microsoft-teams-video-training-4f108e54-240b-4351-8084-b1089f0d21d7. Do all products have such pages? This just won't look as clean in the playlists. It'll give us more flexibility if the option to copy embedded URL from those links. I have also found a few wrong contents in playlists such as in OneNote -> OneNote Quick Start. It displays links to Excel content.

**Comment by dcashpeterson** [comment]
@4Ilan I understand the frustration. You are welcome to use pages like the ones you linked to. The challenge you might encounter is that 1) there is not a page like that for every product so that can create an inconsistent user experience 2) Microsoft is in the process of changing and retiring content so pages that you are currently using today may not be working in the future 3) If you try to pull pages into Learning Pathways you need to ensure that it can be viewed inside an iframe. The first link you provided can be Iframed but it also shows all the chrome and navigation on the page which isn't a great user experience. 

The team at Microsoft are working on an alternative but the wheels are moving very slowly.

**Comment by TracyGH** [comment]
@4Ilan - I had the exact same thought as you. I started adding playlists and assets for missing products or information, sourced from online, public support/training Microsoft resources. 

But then, of course, I considered @dcashpeterson's point: What about when these pages get deprecated, moved, or updated? So, I went a different direction and began building a prototype Q&A agent in Copilot Studio. You can add up to 3 URL knowledge sources but only 2 levels deep, so I added the following 2 Microsoft support/training roots: 

1. https://learn.microsoft.com/en-us/microsoft-365
2. https://support.microsoft.com/en-us/office

So far, I'm pretty impressed with the bot's capabilities and zero human intervention. And I like that it includes a link to the source of its answer. 

![Image](https://github.com/user-attachments/assets/4327139d-d5ac-4c15-8bac-1c506179fd09)

I've had mixed results when enabling the AI general knowledge and generative AI features. At times, it seems to know too much and might answer the question in terms of another software platform or not even in the context of software at all. 

![Image](https://github.com/user-attachments/assets/8c4394c4-682d-4c0d-8c4a-305a80787ef1)

Obviously this solution is still at risk of volatile support sources, but it would certainly be much less labor to update them when that does happen. 

My client is in the public sector so funds are tight, but I ***think*** this is a fairly affordable solution. 
End users don't need a Copilot license, just the folks who build/edit the agent do. But they do charge by message and the quote my client got was $200/month for 25k messages. 

Anywho, just thought I'd share another strategy. 
In some ways the LP feels a bit outdated and perhaps it's time to revisit the approach.

**Comment by pemaenuuuu** [comment]
@jhwang-ms @dcashpeterson 

Hello,

having issues viewing most of the Bookings content under the Teams learning playlist. Was wondering if Bookings is getting its own product playlist?


![Image](https://github.com/user-attachments/assets/fdde3327-cc34-4729-87a5-a013d67b13fe)

**Comment by sogreen81** [comment]
Are there any updates to this issue as I keep seeing post to upload the new version to 5.0 but want to know if this is fixed? We are not able to identify if there are any changes to the learning path information.

**Comment by dcashpeterson** [comment]
@sogreen81 if you are referring to the lack of updates on the content there has been some movement. I am currently working on updating and validating all the assets in Learning Pathways. As you can imagine updating all the assets and playlists across all 10 supported languages is a time consuming process. As this is a volunteer project for me I am doing this as quickly as possible.

If you are referring to some of the bugs that folks identified in version 5 those have been fixed and updated. There are a few outstanding issues. This issue is specific to content challenges.

**Comment by dcashpeterson** [comment]
@sogreen81 this issue is specific to technical issues around content publishing. Your issues are specific to the configuration of the webpart. Please search in the closed issues as many of the things you described are in there and for what is left open a new issue here. This is a community solution and as such Microsoft support does not offer support for it.

**Comment by sogreen81** [comment]
@dcashpeterson  thanks. Do you have a link to the closed issues?

**Comment by dcashpeterson** [comment]
@sogreen81 there is a search box on the issues page. https://github.com/pnp/custom-learning-office-365/issues just change the state:open to state:closed.

**Comment by dcashpeterson** [comment]
Hello all,
We wanted to share some information with you all. We are in the process of checking all of the playlists and all of the assets in all of the languages to ensure that they are still working and removing duplication. As you can imagine this is a long and ongoing process. I have been releasing updates as they are ready behind the scenes. The endpoint has been changed on all assets to a supported one and resolved some of the accessibility challenges. 

If you are looking for new content for a specific topic or product please create an issue (if it isn't already there) and let us know the content from support.microsoft.com you would like to see there. Also, if there are content issues (like broken links) let us know and we can make those changes to ensure that content is up to date.

**Comment by dcashpeterson** [comment]
@clarelcampbell  let me look at the Abandoned showing up in the viewer. That didn't show up in any of our testing. Essentially when items get orphaned because they are no longer associated with a category or sub-category (we deprecated the Yammer playlist) it goes into an abandoned category. That shouldn't be showing in the UI for the viewer. Can you please log that as a separate issue so I can track it outside of content updates.

In the future I will do a better job at documenting the content changes. This was a big one and touched a lot of things. I apologize for the inconvenience. You can hide anything you don't want like "transform your work week" that is Copilot related content.
